views/user/profile.ejs

```diff
@@ -79,9 +79,17 @@
                 </div>
               </div>
               <div class="profile-buttons">
-                <a href="<%=`/user/${user._id}/edit` %>">
-                  <button class="btn btn-primary">Edit Personal Data</button>
-                </a>
+                <% if (user.role === 'recruiter') { %>
+                  <a href="<%=`/user/${user._id}/create_offers` %>" class="btn btn-outline-primary mr-2"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="bi bi-plus" viewBox="1 1 16 16">
+                    <path d="M8 4a.5.5 0 0 1 .5.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3A.5.5 0 0 1 8 4z"/>
+                  </svg> Add job</a>
+                  <a href="<%=`/user/${user._id}/offers` %>" class="btn btn-outline-primary mr-2"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-collection" viewBox="0 0 16 16">
+                    <path d="M2.5 3.5a.5.5 0 0 1 0-1h11a.5.5 0 0 1 0 1h-11zm2-2a.5.5 0 0 1 0-1h7a.5.5 0 0 1 0 1h-7zM0 13a1.5 1.5 0 0 0 1.5 1.5h13A1.5 1.5 0 0 0 16 13V6a1.5 1.5 0 0 0-1.5-1.5h-13A1.5 1.5 0 0 0 0 6v7zm1.5.5A.5.5 0 0 1 1 13V6a.5.5 0 0 1 .5-.5h13a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-.5.5h-13z"/>
+                  </svg>  Your jobs</a>
+                  <% } %>
+                <a href="<%=`/user/${user._id}/edit` %>" class="btn btn-outline-secondary mr-2 br-light"> <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-pencil" viewBox="0 0 16 16">
```

Comment:
I would change this Edit Profile button to "btn-outline-**primary**" because it seems a bit off that they have different colors
![grafik](https://user-images.githubusercontent.com/38247392/103535301-c5757b00-4e90-11eb-9718-cbb17cac01f4.png)